src/components/Headline/docs/Headline.mdx

```diff
@@ -7,20 +7,30 @@ route: /components/headline
 import { Playground } from 'docz';
 import { ItemWrapper } from '../../../../docs/components/ItemWrapper.ts';
 import { Headline } from '../Headline.tsx';
-import { theme } from '../../../essentials/theme.ts';
-import { Colors } from '../../../essentials/Colors/Colors.ts';
+import { HeadlinePropsTable } from './HeadlinePropsTable';
+
 
 # Headline
 
-The Headline component will render an html `h1-6` tag. Please reference the
+The Headline component will render a html `h1-6` tag. Please reference the
```

Comment:
I think `an` in this case is actually correct, but I'm not a native speaker either